src/OrleansRuntime/Placement/DeploymentLoadPublisher.cs

```diff
@@ -178,6 +178,11 @@ private void NotifyAllStatisticsChangeEventsSubscribers(SiloAddress silo, SiloRu
 
         public void SiloStatusChangeNotification(SiloAddress updatedSilo, SiloStatus status)
         {
+            this.ScheduleTask(() => { Utils.SafeExecute(() => this.OnSiloStatusChange(updatedSilo, status), this.logger); }).Ignore();
```

Comment:
this is the other example: when we were actually breaking isolation (as opposite to below, where we were not breaking isolation).
I get it that in theory it was wrong to break isolation. But before I go and try to dig up why we did it this way, I want to ask:
why do you want to change it now? what did not work correctly? What was actually wrong with that?
The data structures here are concurrent and handled correctly I believe, so I don't yet see any reason to change it here, unless you can show me one.